doc/en/example/simple.rst

```diff
@@ -534,23 +534,24 @@ case we just write some informations out to a ``failures`` file::
     import pytest
     import os.path
 
-    @pytest.hookimpl(tryfirst=True)
-    def pytest_runtest_makereport(item, call, __multicall__):
+    @pytest.hookimpl(tryfirst=True, hookwrapper=True)
+    def pytest_runtest_makereport(item, call):
         # execute all other hooks to obtain the report object
-        rep = __multicall__.execute()
+        outcome = yield
+        rep = outcome.get_result()
 
         # we only look at actual failing test calls, not setup/teardown
         if rep.when == "call" and rep.failed:
             mode = "a" if os.path.exists("failures") else "w"
             with open("failures", mode) as f:
                 # let's also access a fixture for the fun of it
-                if "tmpdir" in item.funcargs:
+                if "tmpdir" in item.fixturenames:
```

Comment:
im under the impression we want a naming cleanup for pytest 3.x - funcargs is an inaccurate container name now ^^